python_modules/dagster/dagster/core/workspace/context.py

```diff
@@ -302,7 +302,11 @@ def permissions(self) -> Dict[str, bool]:
         return self._process_context.permissions
 
     def has_permission(self, permission: str) -> bool:
```

Comment:
we should make `has_permission` an abstractmethod on the ABC as well, since it's overridden elsewhere

...also maybe, but this might break mypy
```suggestion
    def has_permission(self, permission: Permissions) -> bool:
```